.gitignore

```diff
@@ -106,4 +106,5 @@ dist
 
 # Environment files
 config.json
+config-test.json
```

Comment:
This doesn't really have to be here since by omitting config.json, the test config.json could be the config.json you have locally.

Reply:
It's in the .gitignore to keep it from floating from dev -> dev. Sure it doesn't matter, but felt keeping the repo clean